internal/persistence/sql/migrations/single_table_test.go

```diff
@@ -0,0 +1,168 @@
+package migrations
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ory/keto/internal/driver"
+	"github.com/ory/keto/internal/driver/config"
+	"github.com/ory/keto/internal/namespace"
+	"github.com/ory/keto/internal/relationtuple"
+	"github.com/ory/keto/internal/x"
+	"github.com/ory/keto/internal/x/dbx"
+)
+
+func TestToSingleTableMigrator(t *testing.T) {
```

Comment:
Am I correct in assuming that we are testing a single namespace here also? Would it make sense to ensure that we properly detect all namespaces if we have multiple namespaces?

Reply:
The function can only migrate a single namespace. We create a new one for each subtest.
The test [below](https://github.com/ory/keto/pull/707/files#diff-d8f4c5835169af823f074a4f9c32416e4bbc979b24a0c37ad1dfdbee095e7ec1R129) does the namespace detection, but only one. Will add multiple ones there.

Reply:
Nice, ty!